### ISSUE TITLE
-Wunused-variable in drivers/mmc/host/dw_mmc.c

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#422]
via kernelCI in linux-next:
```
    2    drivers/mmc/host/dw_mmc.c:239:16: warning: unused variable 'irqflags' [-Wunused-variable]
```

**Comment by nathanchance** [comment]
The patch that causes this has been dropped from the mmc tree and [a v3](https://marc.info/?l=linux-mmc&m=155298689302518&w=2) has been sent to fix this.

(I was about to file this and a few others, you beat me to it lol).

**Comment by nathanchance** [comment]
"Fixed" in `next-20190320` (no longer present because v3 is pending).
